### DISCUSSION TITLE
[Bug report] Unsolved page link in excerpt

### DISCUSSION THREAD

**Discussion by bttrx** [GetPublii/Publii#1907] [Bug Report] [answered]
### Operating system

Linux Mint 22 MATE

### Publii version

0.46.2 (build 16966)

### Issue type

Application

### Bug description

1. Head over to https://www.robertriebisch.de/
2. Look for my post 'Wer Rechtschreibfehler findet, …' (Currently, it is on the front page)
3. In the text the link to 'Impressum' reads `https://www.robertriebisch.de/#non-existing-page-with-id-1`, but it should read `https://www.robertriebisch.de/impressum/`

If you open https://www.robertriebisch.de/posts/2024-12_wer-rechtschreibfehler-findet/ directly, the link to the Impressum page is correct.

**Comment by dziudek** [comment, marked as answer]
@bttrx - thank you for spotting this - it seems that in pages post links works, but in reverse not. I have found a reason and it will be solved in v.0.46.4
